examples/chip-tool/commands/interactive/InteractiveCommands.cpp

```diff
@@ -114,14 +116,18 @@ bool InteractiveStartCommand::ParseCommand(char * command)
             return true;
         }
 
-        args[argsCount++] = token;
-        token             = strtok(nullptr, " ");
+	char* carg = new char[ arg.size() ];
```

Comment:
```suggestion
	char* carg = new char[ arg.size() + 1];
```
no?  Need to leave space for the null-terminator.

Reply:
Right, `c_str()` returns with termination and `strcpy` will copy it.